who_l3_smart_tools/core/terminology/who/terminology.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import pandas as pd
+from who_l3_smart_tools.core.terminology.terminology import ConceptTerminology
+from who_l3_smart_tools.core.terminology.who.schema import HIVConceptSchema
+
+
+class HIVTerminology(ConceptTerminology):
+    """
+    A class representing the HIV terminology.
+
+    This class extends the base ConceptTerminology class and provides specific functionality
+    for working with HIV-related concepts.
+
+    Args:
+        files (list): A list of files containing the HIV terminology data.
+    """
+
+    excel_sheets_prefix = "HIV"
+
+    def __init__(self, files: list):
+        owner_id = "WHO-Smart-Guidelines"
+        super().__init__(owner_id, files, schema=HIVConceptSchema)
+
+    @classmethod
+    def from_excel(cls, file_path: str):
+        directory = os.path.dirname(file_path)
+        files = {}
+        excel_df = pd.read_excel(file_path, sheet_name=None)
+        for sheet_name, df in excel_df.items():
```

Comment:
I don't really love dumping this into files. It's a lot of files for something that should just fit in memory. Is there a reason we can't just operate over the data frame as we do elsewhere in this project?

Reply:
Good point. This decision was because I initially thought this was going to support both csv and xls. I will refactor this to use pyxl on the actual document.